pyglossary/plugins/dict_org.py

```diff
@@ -102,6 +107,23 @@ def close(self) -> None:
 			# self._dictdb.finish()
 			self._dictdb = None
 
+	def compilePatterns(self) -> None:
+		self._re_newline_in_braces = re.compile(r'\{(?P<left>.*?)\n(?P<right>.*?)?\}')
+		self._re_words_in_braces = re.compile(r'\{(?P<word>.+?)\}')
+
+	def prettifyDefinitionText(self, defi: str) -> str:
+                # Handle words in {}. 
+                # First, we remove any \n in {} pairs
+		defi = self._re_newline_in_braces.sub(r'{\g<left>\g<right>}', defi)
+
+                # Then, replace any {words} into <a href="words">words</a>,
+                # so it can be rendered as link correctly
+		defi = self._re_words_in_braces.sub(r'<a href="\g<word>">\g<word></a>', defi)
```

Comment:
You should add a `bword://` prefix to the word as url.
`href="bword://\g<word>`
That's what other plugins do. Please also test it and let me know.